src/wasm.h

```diff
@@ -954,6 +954,9 @@ class Module {
 
   void updateMaps();
 
+  // returns true if the module has a target features section
+  bool readFeatures(FeatureSet&) const;
+
```

Comment:
I feel like this isn't the right place - we've kept `wasm.h` very minimal, leaving utilities to other places. Perhaps `wasm-binary.h` (since feature section is part of the binary format) or `ir/module-utils.h`?